examples/js/effects/VREffect.js

```diff
@@ -19,7 +19,7 @@ THREE.VREffect = function ( renderer, onError ) {
 	var renderRectL, renderRectR;
 	var eyeFOVL, eyeFOVR;
 
-	function gotVRDisplays( displays ) {
```

Comment:
Thanks for the comment.
I'll fix them.

Should we add that to the wiki?
https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style%E2%84%A2